<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="5" passed="3">
  <reporter-output>
  </reporter-output>
<<<<<<< Updated upstream
  <suite name="Default suite" duration-ms="105111" started-at="2019-11-30T12:33:54Z" finished-at="2019-11-30T12:35:39Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="105111" started-at="2019-11-30T12:33:54Z" finished-at="2019-11-30T12:35:39Z">
      <class name="com.torenzo.qa.pomtest.ResetPwdPageTest">
        <test-method status="PASS" signature="launchApp()[pri:0, instance:com.torenzo.qa.pomtest.ResetPwdPageTest@7bedc48a]" name="launchApp" is-config="true" duration-ms="23705" started-at="2019-11-30T18:03:54Z" finished-at="2019-11-30T18:04:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="PASS" signature="LoginIntoAppTest()[pri:0, instance:com.torenzo.qa.pomtest.ResetPwdPageTest@7bedc48a]" name="LoginIntoAppTest" duration-ms="29747" started-at="2019-11-30T18:04:18Z" finished-at="2019-11-30T18:04:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginIntoAppTest -->
        <test-method status="PASS" signature="verifyResetPsdWindow()[pri:0, instance:com.torenzo.qa.pomtest.ResetPwdPageTest@7bedc48a]" name="verifyResetPsdWindow" duration-ms="25072" started-at="2019-11-30T18:04:48Z" finished-at="2019-11-30T18:05:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyResetPsdWindow -->
        <test-method status="PASS" signature="ResetPwdWithInvalidPwd()[pri:0, instance:com.torenzo.qa.pomtest.ResetPwdPageTest@7bedc48a]" name="ResetPwdWithInvalidPwd" duration-ms="6674" started-at="2019-11-30T18:05:13Z" finished-at="2019-11-30T18:05:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ResetPwdWithInvalidPwd -->
        <test-method status="FAIL" signature="ResetPwdWithValidPwd()[pri:0, instance:com.torenzo.qa.pomtest.ResetPwdPageTest@7bedc48a]" name="ResetPwdWithValidPwd" duration-ms="9014" started-at="2019-11-30T18:05:19Z" finished-at="2019-11-30T18:05:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [1234] but found [Your Password was successfully changed.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [1234] but found [Your Password was successfully changed.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.torenzo.qa.pomtest.ResetPwdPageTest.ResetPwdWithValidPwd(ResetPwdPageTest.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ResetPwdWithValidPwd -->
        <test-method status="FAIL" signature="ResetPwdAndLogin()[pri:0, instance:com.torenzo.qa.pomtest.ResetPwdPageTest@7bedc48a]" name="ResetPwdAndLogin" duration-ms="10872" started-at="2019-11-30T18:05:29Z" finished-at="2019-11-30T18:05:39Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[It is not navigated on Home page after clicking on back button expected [Your Password was successfully changed.] but found [Order]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: It is not navigated on Home page after clicking on back button expected [Your Password was successfully changed.] but found [Order]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at com.torenzo.qa.pomtest.ResetPwdPageTest.ResetPwdAndLogin(ResetPwdPageTest.java:126)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ResetPwdAndLogin -->
      </class> <!-- com.torenzo.qa.pomtest.ResetPwdPageTest -->
=======
  <suite name="Default suite" duration-ms="204109" started-at="2019-12-02T05:36:52Z" finished-at="2019-12-02T05:40:17Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="204109" started-at="2019-12-02T05:36:52Z" finished-at="2019-12-02T05:40:17Z">
      <class name="com.torenzo.qa.pomtest.NCKOTOrderPageTest">
        <test-method status="PASS" signature="launchApp()[pri:0, instance:com.torenzo.qa.pomtest.NCKOTOrderPageTest@37ceb1df]" name="launchApp" is-config="true" duration-ms="11472" started-at="2019-12-02T11:06:52Z" finished-at="2019-12-02T11:07:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="PASS" signature="verfiyHomePageTest()[pri:0, instance:com.torenzo.qa.pomtest.NCKOTOrderPageTest@37ceb1df]" name="verfiyHomePageTest" duration-ms="35563" started-at="2019-12-02T11:07:04Z" finished-at="2019-12-02T11:07:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verfiyHomePageTest -->
        <test-method status="PASS" signature="paymentPageTest()[pri:0, instance:com.torenzo.qa.pomtest.NCKOTOrderPageTest@37ceb1df]" name="paymentPageTest" duration-ms="102061" started-at="2019-12-02T11:07:40Z" finished-at="2019-12-02T11:09:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- paymentPageTest -->
        <test-method status="PASS" signature="editPageTest()[pri:0, instance:com.torenzo.qa.pomtest.NCKOTOrderPageTest@37ceb1df]" name="editPageTest" duration-ms="33836" started-at="2019-12-02T11:09:22Z" finished-at="2019-12-02T11:09:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- editPageTest -->
        <test-method status="PASS" signature="editPageTestNegative()[pri:0, instance:com.torenzo.qa.pomtest.NCKOTOrderPageTest@37ceb1df]" name="editPageTestNegative" duration-ms="21142" started-at="2019-12-02T11:09:55Z" finished-at="2019-12-02T11:10:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- editPageTestNegative -->
      </class> <!-- com.torenzo.qa.pomtest.NCKOTOrderPageTest -->
>>>>>>> Stashed changes
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
